java/ql/src/utils/flowtestcasegenerator/GenerateFlowTestCase.py

```diff
@@ -127,7 +131,13 @@ def qualifiedOuterNameFromCsvRow(row):
 os.makedirs(queryDir)
 qlFile = os.path.join(queryDir, "gen.ql")
 with open(os.path.join(queryDir, "qlpack.yml"), "w") as f:
-    f.write("name: test-generation-query\nversion: 0.0.0\nlibraryPathDependencies: codeql/java-queries")
+    f.write(f"""name: test-generation-query
```

Comment:
Why is this a format string? It doesn't seem to use any format arguments.

Reply:
You are right - there is no need.